Packs/Microsoft365Defender/Integrations/Microsoft365DefenderEventCollector/Microsoft365DefenderEventCollector.yml

```diff
@@ -56,41 +69,53 @@ configuration:
   hidden: false
   name: verify
   required: false
-  defaultvalue: true
+  defaultvalue: 'true'
```

Comment:
```suggestion
  defaultvalue: true
```

Reply:
This chahnge made by the sdk format 
without this it will fail on  IN152 error